api/Navigator.json

```diff
@@ -2365,6 +2365,112 @@
           }
         }
       },
+      "unregisterProtocolHandler": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "38"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": [
+              {
+                "version_added": "25"
+              },
+              {
+                "version_added": "≤12.1",
```

Comment:
Perhaps "11.6" if `registerProtocolHandler` data can be trusted.

Reply:
Disregard this, the two were added at different times in Chrome and the same might have happened in Presto.